src/utils/mathJaxHelpers.ts

```diff
@@ -0,0 +1,18 @@
+interface MathJax {
+  typesetPromise: () => Promise<unknown>;
+}
+
+interface MathJaxWindow extends Window {
+  MathJax?: MathJax;
+}
+
+export const typesetMathJax = () => {
+  const mathJaxWindow = window as MathJaxWindow;
+  if (mathJaxWindow.MathJax) {
+    mathJaxWindow.MathJax.typesetPromise();
+  }
+};
+
+export const stringIncludesMathJax = (input = '') => {
+  return input.includes('$');
```

Comment:
Bruke regex i stedet? vil gi færre false positives, men vil også koste mer for hver kjøring... tipper det vil koste mer i sum